src/plugins/kibana_usage_collection/server/collectors/kibana/get_saved_object_counts.test.ts

```diff
@@ -46,9 +66,9 @@ describe('getSavedObjectsCounts', () => {
           ],
         },
       },
-    }));
+    });
 
-    const results = await getSavedObjectsCounts(callCluster as any, '.kibana');
+    const results = await getSavedObjectsCounts(esClient as any, '.kibana');
```

Comment:
NIT: No need for the `as any`
```suggestion
    const results = await getSavedObjectsCounts(esClient, '.kibana');
```